driver/pom.xml

```diff
@@ -3,16 +3,16 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.michaelpollmeier</groupId>
     <artifactId>orientdb-gremlin</artifactId>
-    <version>3.2.1.2-SNAPSHOT</version>
+    <version>3.2.2.0</version>
```

Comment:
maybe i was a bit optimistic that this would work straight away :)

On 14/09/16 07:14, Marvin Froeder wrote:
> In driver/pom.xml
> <https://github.com/mpollmeier/orientdb-gremlin/pull/94#discussion_r78625242>:
>
>> @@ -3,16 +3,16 @@
>>      <modelVersion>4.0.0</modelVersion>
>>      <groupId>com.michaelpollmeier</groupId>
>>      <artifactId>orientdb-gremlin</artifactId>
>> -    <version>3.2.1.2-SNAPSHOT</version>
>> +    <version>3.2.2.0</version>
>
> Missing -SNAPSHOT?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mpollmeier/orientdb-gremlin/pull/94/files/fe6287348e4ec62119871a565bc65a8eda6958b3#r78625242>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAe7gBUgKhbqYefShR9lHGgaCM6gTyosks5qpvYPgaJpZM4J7q9S>.
>